packages/block-editor/src/components/font-sizes/fluid-utils.js

```diff
@@ -215,7 +289,8 @@ export function getTypographyValueAndUnit( rawValue, options = {} ) {
  * @return {number|undefined} Value rounded to standard precision.
  */
 export function roundToPrecision( value, digits = 3 ) {
+	const base = Math.pow( 10, digits + 1 );
 	return Number.isFinite( value )
-		? parseFloat( value.toFixed( digits ) )
+		? parseFloat( ( value + 1 / base ).toFixed( digits ) )
```

Comment:
It looks like this adds a `1` at a single decimal place smaller than the rounding digits. I take it this is to ensure we work around weird floating point precision issues? I see a slightly different approach has been taken elsewhere in the repo earlier (e.g. in [the performance reporter](https://github.com/WordPress/gutenberg/blob/45cffa2beaedd9146944f3a084b3748ce5572362/packages/e2e-tests/config/performance-reporter.js#L12)), but this one looks good to me.

Reply:
Yes, it was to ensure we get 3 all the time, even when JS randomly throws `0.000001` at us :D 

Thanks for linking to the other one. It should work as well. Might be a good opportunity later to extract the util